assets/components/dough_theme/form/_form.scss

```diff
@@ -92,3 +92,27 @@
 .form__list {
   @include body(16, 24);
 }
+
+.form__custom-select {
+  width: 250px;
+  // This is copied from budget planner
+  height: 30px; // custom height
+  border: 1px solid; // $greyscale4; colour is a pacs variable
+  padding: 0 8px;
+  background: #fff right 10px center no-repeat;
+  background-image: image-url('dough/assets/images/select-arrow.png');
+  background-size: 15px 7px;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  // Pacs custom
+  // font-weight: 400;
+
+  .svg & {
+    background-image: image-url('dough/assets/images/icon_sprite.svg#svg-icon--select-arrow');
```

Comment:
what would happen here if the latest dough has not been added to a project as well? should we be considering a fallback for this or not? wdyt?

Reply:
Another good question. I think it highlights the weakness of separating the location of the image (Dough) from the location of the styles (Yeast). Trouble is I don't think it's possible to store images on Yeast. But yes, if the latest version of Dough wasn't loaded on the project the image would not render, though this wouldn't be a problem when it is mounted on Frontend. 

I'm also uneasy about over-riding browser native styles for form elements like ```<select>``` TBH but that has already been done in Budget Planner so I guess it's too late to worry about it. Any fallback should really be the native style I suppose but we're specifically removing that here. I don't think there's a foolproof way round this if it's something we want to do.